spec/features/restaurants/visitor_tries_update_restaurant_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'prevent unuathorized updates to restaurant' do
+  context 'as a visitor' do
+    scenario 'I cannot save a restaurant' do
+      visit new_restaurant_path
+      expect(page).to have_content("Log in")
```

Comment:
I'm not sure I understand this test. You should expect to not see the ability to save content or something like that than just a login screen.